
const handleGetAlluser = async(req,res)=>{
    try {
        
    } catch (error) {
        console.log("Error",error);
        res.status(500).json({message:"Internal Server Error"})
    }
}
const handleCreateUser = async(req,res)=>{
    try {
        
    } catch (error) {
        console.log("Error",error);
        res.status(500).json({message:"Internal Server Error"})
    }
}
const handleUpdateUser = async(req,res)=>{
    try {
        
    } catch (error) {
        console.log("Error",error);
        res.status(500).json({message:"Internal Server Error"})
    }
}
const handleDeleteUser = async(req,res)=>{
    try {
        
    } catch (error) {
        console.log("Error",error);
        res.status(500).json({message:"Internal Server Error"})
    }
}


module.exports = {handleCreateUser,handleGetAlluser,handleUpdateUser,handleDeleteUser}